src/UI/SlotArea.cpp

```diff
@@ -996,14 +996,14 @@ void cSlotAreaAnvil::OnTakeResult(cPlayer & a_Player)
 	m_MaximumCost = 0;
 	static_cast<cAnvilWindow &>(m_ParentWindow).SetRepairedItemName("", nullptr);
 
-	int PosX, PosY, PosZ;
-	static_cast<cAnvilWindow &>(m_ParentWindow).GetBlockPos(PosX, PosY, PosZ);
+	Vector3i Pos;
+	static_cast<cAnvilWindow &>(m_ParentWindow).GetBlockPos(Pos);
 
 	BLOCKTYPE Block;
 	NIBBLETYPE BlockMeta;
-	a_Player.GetWorld()->GetBlockTypeMeta(PosX, PosY, PosZ, Block, BlockMeta);
+	a_Player.GetWorld()->GetBlockTypeMeta(Pos, Block, BlockMeta);
 
-	const Vector3i BlockPos{PosX, PosY, PosZ};
+	const Vector3i BlockPos{Pos};
```

Comment:
This copy is not needed now, just rename `Pos` to `BlockPos` in line 999, then use it everywhere.